Sources/SwiftSyntaxBuilder/SyntaxNodeWithBody.swift

```diff
@@ -37,7 +55,7 @@ public protocol HasTrailingCodeBlock {
 }
 
 public extension HasTrailingCodeBlock where Self: SyntaxExpressibleByStringInterpolation {
```

Comment:
There's a TODO just above this that you could remove now if you want

Reply:
Good point. Removed it